### DISCUSSION TITLE
Laravel Livewire Forms - Possible Error In Docs

### DISCUSSION THREAD

**Discussion by PHPikachu** [livewire/livewire#8319] [1. Help]
Hi all,

I'm working with Livewire V3's forms currently. I'm following the example on the [site](https://livewire.laravel.com/docs/forms#extracting-a-form-object) below:

> Here's what it would look like to use this same form object for an UpdatePost component and fill it with initial data:
> 
> ```
> <?php
>  
> namespace App\Livewire\Forms;
>  
> use Livewire\Attributes\Validate;
> use Livewire\Form;
> use App\Models\Post;
>  
> class PostForm extends Form
> {
>     public ?Post $post;
>  
>     #[Validate('required|min:5')]
>     public $title = '';
>  
>     #[Validate('required|min:5')]
>     public $content = '';
>  
>     public function setPost(Post $post)
>     {
>         $this->post = $post;
>  
>         $this->title = $post->title;
>  
>         $this->content = $post->content;
>     }
>  
>     public function store()
>     {
>         $this->validate();
>  
>         Post::create($this->only(['title', 'content']));
>     }
>  
>     public function update()
>     {
>         $this->post->update(
>             $this->all()
>         );
>     }
> }
> ```

When calling `update()` - it errors with:

> SQLSTATE[42S22]: Column not found: 1054 Unknown column 'post' in 'field list'

Which is caused by the `$this->all()` within the update function, it's including the `public ?Post $post` in the update.

Is this an error in the documentation? Should it instead be `$this->except(['post'])` or something similar?

Thanks!

**Comment by danie-ramdhani** [comment]
yes it is, already discussed in https://github.com/livewire/livewire/discussions/8158

**Reply by PHPikachu** [reply]
Apologies, I didn't look hard enough clearly! Thank you so much for pointing this out 😄
